src/pybind/mgr/dashboard/controllers/rbd.py

```diff
@@ -593,3 +609,84 @@ def remove_image(self, pool_name, group_name, image_name, namespace=None):
             RbdService.validate_namespace(ioctx, namespace)
             ioctx.set_namespace(namespace)
             return group.remove_image(ioctx, image_name)
+
+
+@APIRouter('/block/pool/{pool_name}/group/{group_name}/snap', Scope.RBD_IMAGE)
+@APIDoc("RBD Group Snapshot Management API", "RbdGroupSnapshot")
+class RbdGroupSnapshot(RESTController):
+
+    RESOURCE_ID = "snapshot_name"
+
+    def __init__(self):
+        super().__init__()
+        self.rbd_inst = rbd.RBD()
+
+    @handle_rbd_error()
+    @EndpointDoc("List group snapshots",
+                 parameters={
+                     'pool_name': (str, 'Name of the pool'),
+                     'group_name': (str, 'Name of the group'),
+                 },
+                 responses={200: RBD_GROUP_SNAPSHOT_LIST_SCHEMA})
+    def list(self, pool_name, group_name, namespace=None):
```

Comment:
Very nitpicky, and for this particular case is not as important but even though it's python we try to give type definitions.


```suggestion
    def list(self, pool_name: str, group_name: str, namespace: str=None):
```